paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButtonNew.kt

```diff
@@ -73,7 +72,7 @@ private const val CENTER_ALIGNED = 0f
 internal sealed interface PrimaryButtonProcessingState {
     data class Idle(val error: ResolvableString?) : PrimaryButtonProcessingState
```

Comment:
We haven't used `error` in the `PrimaryButton` definition and I haven't seen a use case for it. Should we remove it?

Reply:
Yep, let’s remove it!